great_expectations/rule_based_profiler/data_assistant_result/onboarding_data_assistant_result.py

```diff
@@ -8,7 +8,7 @@
 
 class OnboardingDataAssistantResult(DataAssistantResult):
     @property
-    def metric_expectation_map(self) -> Dict[Union[str, Tuple[str]], str]:
+    def metric_expectation_map(self) -> Dict[Union[str, Tuple[str, ...]], str]:
```

Comment:
There is also apparently the formal type `Ellipsis` that translates to the three dots, but what is the purpose of it here?  Thanks.

Reply:
https://docs.python.org/3/library/typing.html#typing.Tuple